src/test/java/org/opentripplanner/transit/raptor/speed_test/testcase/Result.java

```diff
@@ -20,7 +20,7 @@
  * itinerary. The result can be expected or actual, both represented by this class.
  */
 class Result {
-    private static final Pattern STOPS_PATTERN = Pattern.compile(" ~ (\\d+) ~ ");
+    private static final Pattern STOPS_PATTERN = Pattern.compile(" ~ (\\.+) ~ ");
```

Comment:
This does not look right. I think the pattern matches " ~ .  ~ ", " ~ ..  ~ ", " ~ ...  ~ ", and so on. I am not sure we can use a regular expression here any more. In stead of parsing the path string we should probably keep an explicit list of stops in the result.